nemoguardrails/colang/v2_x/runtime/statemachine.py

```diff
@@ -2422,5 +2422,7 @@ def _is_child_activated_flow(state: State, flow_state: FlowState) -> bool:
     return (
         flow_state.activated > 0
         and flow_state.parent_uid is not None
+        and flow_state.parent_uid
+        in state.flow_states  # TODO: Figure out why this can fail sometimes
```

Comment:
Nitpick: the `in` on another line makes it a bit harder to read the condition.

Reply:
Yeah, agreed. The autoformatting is to blame!